shell-completion/bash/resolvectl

```diff
@@ -25,8 +25,8 @@ __contains_word () {
 }
 
 __get_interfaces(){
-        { cd /sys/class/net && echo *; } | \
-        while read -d' ' -r name; do
+        local name
+        for name in $(cd /sys/class/net && ls); do
```

Comment:
This looks OK, but the commit message doesn't mention what was wrong with the previous version?